frontend/src/components/widgets/Multiselect/Multiselect.test.tsx

```diff
@@ -288,4 +288,116 @@ describe("Multiselect widget", () => {
       }
     )
   })
+  describe("properly invalidates going over max selections", () => {
+    it("has correct noResultsMsg when maxSelections is not passed", () => {
+      const props = getProps(
+        MultiSelectProto.create({
+          id: "1",
+          label: "Label",
+          default: [0],
+          options: ["a", "b", "c"],
+        })
+      )
+      const wrapper = mount(<Multiselect {...props} />)
+
+      expect(
+        wrapper
+          .find(UISelect)
+          .prop("noResultsMsg")
+          ?.toString()
```

Comment:
nits (here and below):

* We don't need to use the `?` operator here since the `"noResultsMsg"` prop should be found, and the test should blow up if not (it'll blow up anyway, but there's no reason to use optional chaining as far as I can tell)
* Similarly, shouldn't `"noResultsMsg"` already be a string? I don't think the `toString()` here is needed
* [toHaveProperty()](https://jestjs.io/docs/expect#tohavepropertykeypath-value) is a bit more semantically clear here since we don't need to do any processing on the prop after getting it